airflow/models/taskinstance.py

```diff
@@ -2502,6 +2510,7 @@ def signal_handler(signum, frame):
             # Execute the task
             with set_current_context(context):
                 result = self._execute_task(context, task_orig)
+                self.rendered_map_index = jinja_env.from_string(context["map_index_template"]).render(context)
```

Comment:
I don’t think `context["map_index_template"]` is a thing?

Reply:
I think it should be, at least in this proposed implementation. If DAG authors are to do something like...

```
    @task(map_index_template="{{ my_template }}")
    def my_task(my_template, **kwargs):
        context = get_current_context()
        context.update({"my_template": my_template})

    my_task.expand(my_template=["abc", "def"])
```

Then this is how it will get rendered. Please let me know if I'm misunderstanding, though.

Reply:
Ah OK. Sounds good to me.